app/validators/candidate_interface/continuous_applications/course_selection_validator.rb

```diff
@@ -4,24 +4,40 @@ module ContinuousApplications
     # Conditional exists to remove the course that is being edited from the checks for duplication
     class CourseSelectionValidator < ActiveModel::Validator
```

Comment:
I think we could rename this to `DuplicateCourseValidator`

Reply:
Sure, lets do that in another PR